.gitignore

```diff
@@ -66,3 +65,7 @@ htmlcov/
 # testing: parts of old trulens_eval package for backwards compatibility testing
 _trulens_eval/
 data/
+
+# Staged artifacts for snowflake connector
+!src/connectors/snowflake/trulens/data
```

Comment:
is this necessary to add? Doesn't this say to not ignore `src/connectors/snowflake/trulens/data`? But no one was ignoring it to begin with.

Reply:
i had to add it there for some reason. iirc the `data/` rule above is ignoring it

Reply:
hmm could we verify this? 

`data/` should not be ignoring `src/connectors/snowflake/trulens/data` (`**/data/`, OTTH, should)